components/resources/brave_components_strings.grd

```diff
@@ -1006,6 +1006,7 @@
       <part file="speedreader_strings.grdp" />
       <part file="wallet_strings.grdp" />
       <part file="tor_strings.grdp" />
+      <part file="brave_vpn_strings.grdp" />
```

Comment:
seems like we should wrap this line with the conditional, instead of in the file?